sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -188,11 +203,31 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     }
   }
 
+  // This will abort any pending request in the IdentityClient
+  abortRequests(): void {
+    let controller: AbortController | undefined;
+    while ((controller = this.abortControllers.shift())) {
```

Comment:
nit: I feel like `forEach` or for loop would be clearer followed by setting `this.abortControllers` to an empty array. Unless there's a reason to remove them one at a time

Reply:
For concurrency reasons it might be better to remove them one at a time 🤔